### ISSUE TITLE
No sudo

### ISSUE THREAD

**Issue by jacobtomlinson** [pangeo-data/pangeo-stacks#12]
Not sure if this is an intentional thing but sudo has disappeared compared to the old images.

**Comment by jhamman** [comment]
This is a known feature? of repo2docker. This is also the same reason there are problems modifying a conda environment from the new images (#13). I think we can hack around it using the appendix in this repo.

**Comment by yuvipanda** [comment]
Yeah, could do this with an appendix if needed. Installations of JupyterHub / BinderHub can then use kubernetes security policies to restrict use of it if needed.
